listenbrainz/webserver/views/recommendations_cf_recording.py

```diff
@@ -92,13 +92,17 @@ def _get_template(active_section, user):
             error_msg="An error occurred while processing your request. Check back later!"
         )
 
-    spotify_data = spotify.get_user_dict(current_user.id)
+    spotify_data = {}
+    current_user_data = {}
 
-    current_user_data = {
-            "id": current_user.id,
-            "name": current_user.musicbrainz_id,
-            "auth_token": current_user.auth_token,
-    }
+    if current_user:
```

Comment:
I think this will work `current_user.is_authenticated`

Reply:
yep